### PR TITLE
Create README.hi.md

### PR DESCRIPTION
**Pull request by anurag01a**

### CODE REVIEW

**Comment by jlooper** [comment]
Is this ok to merge?

**Comment by Anurag-0-1-A** [comment]
Reviewers are yet to review it.

@innovation-platform , @AdityaGarg00 & @EliteWarrior315 ,Kindly review.

File: 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md [hunk 1]
```diff
@@ -0,0 +1,266 @@
...
+
+## व्याख्यान के बाद प्रश्नोत्तरी
+
+[व्याख्यान के बाद प्रश्नोत्तरी](https://brave-island-0b7c7f50f.azurestaticapps.net/quiz/4)
+
+## समीक्षा और आत्म अध्ययन
+
+* Arduino प्लेटफॉर्म के बारे में अधिक समझने के लिए [Arduino गेटिंग स्टार्ट गाइड](https://www.arduino.cc/en/Guide/Introduction) पढ़ें।
+* रास्पबेरी पाई के बारे में अधिक जानने के लिए [रास्पबेरी पाई 4 का परिचय](https://www.raspberrypi.org/products/raspberry-pi-4-model-b/) पढ़ें।
+* [इलेक्ट्रिकल इंजीनियरिंग जर्नल में सीपीयू, एमपीयू, एमसीयू और जीपीयू के अक्सर पूछे जाने वाले सवाल क्या हैं](https://www.eejournal.com/article/what-the-faq) में कुछ अवधारणाओं और शब्दों के बारे में और जानें। -are-cpus-mpus-mcus-and-gpus/)।
```
Comment (AdityaGarg00) [inline 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md:260]: check the link in this line

**Review by AdityaGarg00 — CHANGES_REQUESTED** [review]

File: 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md [hunk 2]
```diff
@@ -0,0 +1,266 @@
...
+
+[व्याख्यान पूर्व प्रश्नोत्तरी](https://brave-island-0b7c7f50f.azurestaticapps.net/quiz/3)
+
+## परिचय
+
+यह पाठ पिछले पाठ में शामिल कुछ अवधारणाओं में गहराई से गोता लगाता है।
+
+इस पाठ में हम कवर करेंगे:
+
+* [एक IoT एप्लिकेशन के अंश](#एक-IoT-एप्लिकेशन-के-अंश)
```
[the remark this answers is not in the document]
Comment (Anurag-0-1-A) [reply 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md:25]: I am also not getting why the link is not working.

[the remark this answers is not in the document]
Comment (Anurag-0-1-A) [reply 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md:25]: It is the index of the topics...

Upon clicking them will take you to that respective topic 

i.e माइक्रोकंट्रोलर्स में गहराई से गोता लगाएँ will take you to Line 81

while सिंगल-बोर्ड कंप्यूटरों में गहरा गोता लगाएँ will take you to Line 201

**Review by ArnoldJohn0495 — APPROVED** [review]

**Comment by jimbobbennett** [comment]
@innovation-platform , @AdityaGarg00 & @EliteWarrior315 ,Kindly review.

File: 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md [hunk 3]
```diff
@@ -0,0 +1,266 @@
...
+रास्पबेरी पाई की जांच करें।
+
+यदि आप इन पाठों के लिए रास्पबेरी पाई का उपयोग कर रहे हैं, तो बोर्ड पर विभिन्न हार्डवेयर घटकों के बारे में पढ़ें।
+
+* आप [रास्पबेरी पाई हार्डवेयर डॉक्यूमेंटेशन पेज](https://www.raspberrypi.org/documentation/hardware/raspberrypi/) पर इस्तेमाल किए गए प्रोसेसर के बारे में जानकारी पा सकते हैं। आप जिस पाई का उपयोग कर रहे हैं उसमें प्रयुक्त प्रोसेसर के बारे में पढ़ें।
+* GPIO पिन का पता लगाएँ। [रास्पबेरी पाई GPIO दस्तावेज़ीकरण](https://www.raspberrypi.org/documentation/hardware/raspberrypi/gpio/README.md) पर उनके बारे में और पढ़ें। अपने पाई पर अलग-अलग पिन की पहचान करने के लिए [GPIO पिन यूसेज गाइड](https://www.raspberrypi.org/documentation/usage/gpio/README.md) का इस्तेमाल करें।
+
+### प्रोग्रामिंग सिंगल-बोर्ड कंप्यूटर
+
+सिंगल-बोर्ड कंप्यूटर पूर्ण कंप्यूटर हैं, जो एक पूर्ण OS चला रहे हैं। इसका मतलब यह है कि माइक्रोकंट्रोलर के विपरीत प्रोग्रामिंग भाषाओं, ढांचे और टूल की एक विस्तृत श्रृंखला है जिसका उपयोग आप उन्हें कोड करने के लिए कर सकते हैं, जो Arduino जैसे ढांचे में बोर्ड के समर्थन पर भरोसा करते हैं। अधिकांश प्रोग्रामिंग भाषाओं में पुस्तकालय होते हैं जो सेंसर और एक्चुएटर्स से डेटा भेजने और प्राप्त करने के लिए GPIO पिन तक पहुंच सकते हैं।
```
Comment (EliteWarrior315) [inline 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md:236]:
```suggestion
सिंगल-बोर्ड कंप्यूटर पूर्ण कंप्यूटर हैं, जो एक पूर्ण OS चला रहे हैं। इसका मतलब यह है कि माइक्रोकंट्रोलर के विपरीत, प्रोग्रामिंग भाषाओं, ढांचे और टूल की एक विस्तृत श्रृंखला है जिसका उपयोग आप उन्हें कोड करने के लिए कर सकते हैं, जो Arduino जैसे ढांचे के विपरीत है जो Arduino में बोर्ड के समर्थन पर भरोसा करते हैं। अधिकांश प्रोग्रामिंग भाषाओं में लाइब्रेरीज़ होते हैं जो सेंसर और एक्चुएटर्स से डेटा भेजने और प्राप्त करने के लिए GPIO पिन तक पहुंच सकते हैं।
```

File: 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md [hunk 4]
```diff
@@ -0,0 +1,266 @@
...
+
+✅ आप किन प्रोग्रामिंग भाषाओं से परिचित हैं? क्या वे लिनक्स पर समर्थित हैं?
+
+रास्पबेरी पाई पर IoT अनुप्रयोगों के निर्माण के लिए सबसे आम प्रोग्रामिंग भाषा पायथन है। पाई के लिए डिज़ाइन किए गए हार्डवेयर का एक विशाल पारिस्थितिकी तंत्र है, और इनमें से लगभग सभी में संबंधित कोड शामिल हैं जो उन्हें पायथन पुस्तकालयों के रूप में उपयोग करने के लिए आवश्यक हैं। इनमें से कुछ पारिस्थितिक तंत्र 'टोपी' पर आधारित होते हैं - तथाकथित क्योंकि वे एक टोपी की तरह पाई के ऊपर बैठते हैं और एक बड़े सॉकेट से 40 GPIO पिन से जुड़ते हैं। ये टोपियां अतिरिक्त क्षमताएं प्रदान करती हैं, जैसे स्क्रीन, सेंसर, रिमोट-नियंत्रित कार, या एडेप्टर आपको मानकीकृत केबल के साथ सेंसर प्लग-इन करने की अनुमति देते हैं
```
Comment (EliteWarrior315) [inline 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md:240]:
```suggestion
रास्पबेरी पाई पर IoT अनुप्रयोगों के निर्माण के लिए सबसे आम प्रोग्रामिंग भाषा पायथन है। पाई के लिए डिज़ाइन किए गए हार्डवेयर का एक विशाल पारिस्थितिकी तंत्र है, और इनमें से लगभग सभी में संबंधित कोड शामिल हैं जो उन्हें पायथन पुस्तकालयों के रूप में उपयोग करने के लिए आवश्यक हैं। इनमें से कुछ पारिस्थितिक तंत्र 'टोपी' पर आधारित होते हैं - तथाकथित क्योंकि वे एक टोपी की तरह पाई के ऊपर बैठते हैं और एक बड़े सॉकेट से 40 GPIO पिन से जुड़ते हैं। ये टोपियां अतिरिक्त क्षमताएं प्रदान करती हैं, जैसे स्क्रीन, सेंसर, रिमोट-नियंत्रित कार, या मानकीकृत केबल वाले सेंसर प्लग-इन करना।
```

File: 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md [hunk 5]
```diff
@@ -0,0 +1,266 @@
...
+
+### पेशेवर IoT परिनियोजन में सिंगल-बोर्ड कंप्यूटर का उपयोग
+
+एकल-बोर्ड कंप्यूटर का उपयोग पेशेवर IoT परिनियोजन के लिए किया जाता है, न कि केवल डेवलपर किट के रूप में। वे हार्डवेयर को नियंत्रित करने और मशीन लर्निंग मॉडल चलाने जैसे जटिल कार्यों को चलाने के लिए एक शक्तिशाली तरीका प्रदान कर सकते हैं। उदाहरण के लिए, एक [रास्पबेरी पाई 4 कंप्यूट मॉड्यूल](https://www.raspberrypi.org/blog/raspberry-pi-compute-module-4/) है जो रास्पबेरी पाई 4 की सारी शक्ति प्रदान करता है लेकिन एक में अधिकांश पोर्ट के बिना कॉम्पैक्ट और सस्ता फॉर्म फैक्टर, जिसे कस्टम हार्डवेयर में स्थापित करने के लिए डिज़ाइन किया गया है।
```
Comment (EliteWarrior315) [inline 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md:244]:
```suggestion
सिंहले-बोर्ड कंप्यूटर का उपयोग पेशेवर IoT परिनियोजन के लिए किया जाता है, न कि केवल डेवलपर किट के रूप में। वे हार्डवेयर को नियंत्रित करने और मशीन लर्निंग मॉडल चलाने जैसे जटिल कार्यों को चलाने के लिए एक शक्तिशाली तरीका प्रदान कर सकते हैं। उदाहरण के लिए, एक [रास्पबेरी पाई 4 कंप्यूट मॉड्यूल](https://www.raspberrypi.org/blog/raspberry-pi-compute-module-4/) है जो रास्पबेरी पाई 4 की सारी शक्ति प्रदान करता है लेकिन एक में अधिकांश पोर्ट के बिना सुगठित और सस्ता रूप गुणक, जिसे कस्टम हार्डवेयर में स्थापित करने के लिए डिज़ाइन किया गया है।
```

File: 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md [hunk 6]
```diff
@@ -0,0 +1,266 @@
...
+
+---
+
+## 🚀चुनौती
+
+पिछले पाठ में चुनौती थी कि आप अपने घर, स्कूल या कार्यस्थल में जितने हो सके उतने IoT उपकरणों को सूचीबद्ध करें। इस सूची में प्रत्येक डिवाइस के लिए, क्या आपको लगता है कि वे माइक्रोकंट्रोलर या सिंगल-बोर्ड कंप्यूटर, या यहां तक ​​कि दोनों के मिश्रण के आसपास बनाए गए हैं?
```
Comment (EliteWarrior315) [inline 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md:250]:
```suggestion
पिछले पाठ में चुनौती थी कि आप अपने घर, स्कूल या कार्यस्थल में जितने हो सके उतने IoT उपकरणों को सूचीबद्ध करें। इस सूची में प्रत्येक डिवाइस के लिए, आपको क्या लगता है कि वे माइक्रोकंट्रोलर या सिंगल-बोर्ड कंप्यूटर, या दोनों के मिश्रण के आसपास बनाए गए हैं?
```

File: 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md [hunk 7]
```diff
@@ -0,0 +1,266 @@
...
+
+[व्याख्यान के बाद प्रश्नोत्तरी](https://brave-island-0b7c7f50f.azurestaticapps.net/quiz/4)
+
+## समीक्षा और आत्म अध्ययन
+
+* Arduino प्लेटफॉर्म के बारे में अधिक समझने के लिए [Arduino गेटिंग स्टार्ट गाइड](https://www.arduino.cc/en/Guide/Introduction) पढ़ें।
+* रास्पबेरी पाई के बारे में अधिक जानने के लिए [रास्पबेरी पाई 4 का परिचय](https://www.raspberrypi.org/products/raspberry-pi-4-model-b/) पढ़ें।
+* [इलेक्ट्रिकल इंजीनियरिंग जर्नल में सीपीयू, एमपीयू, एमसीयू और जीपीयू के अक्सर पूछे जाने वाले सवाल क्या हैं](https://www.eejournal.com/article/what-the-faq-are-cpus-mpus-mcus-and-gpus/) में कुछ अवधारणाओं और शब्दों के बारे में और जानें। 
+
+✅आप किस हार्डवेयर प्लेटफॉर्म का उपयोग करना चाहते हैं, या यदि आप इसके बजाय यह तय करने के लिए [हार्डवेयर गाइड](https://github.com/microsoft/IoT-For-Beginners/blob/main/hardware.md) में लिंक का पालन करके दिखाए गए लागतों के साथ इन गाइड का उपयोग करें वर्चुअल डिवाइस का उपयोग करें।
```
Comment (EliteWarrior315) [inline 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md:262]:
```suggestion
✅आप किस हार्डवेयर प्लेटफॉर्म का उपयोग करना चाहते हैं, या यदि आप वर्चुअल डिवाइस का उपयोग करना चाहते हैं, तो यह तय करने के लिए [हार्डवेयर गाइड](https://github.com/microsoft/IoT-For-Beginners/blob/main/hardware.md) में दिए गए लिंक का पालन करके दिखाए गए लागतों के साथ इन गाइडों का उपयोग करें।

File: 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md [hunk 8]
```diff
@@ -0,0 +1,266 @@
...
+
+## कार्यभार
+
+[माइक्रोकंट्रोलर और सिंगल-बोर्ड कंप्यूटरों की तुलना और तुलना करें](https://github.com/microsoft/IoT-For-Beginners/blob/main/1-getting-started/lessons/2-deeper-dive/assignment.md)
```
Comment (EliteWarrior315) [inline 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md:266]:
```suggestion
[माइक्रोकंट्रोलर और सिंगल-बोर्ड कंप्यूटरों की तुलना करें](https://github.com/microsoft/IoT-For-Beginners/blob/main/1-getting-started/lessons/2-deeper-dive/assignment.md)
```

File: 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md [hunk 9]
```diff
@@ -0,0 +1,266 @@
...
+
+[व्याख्यान पूर्व प्रश्नोत्तरी](https://brave-island-0b7c7f50f.azurestaticapps.net/quiz/3)
+
+## परिचय
+
+यह पाठ पिछले पाठ में शामिल कुछ अवधारणाओं में गहराई से अध्ययन करता है।
+
+इस पाठ में हम कवर करेंगे:
+
+* [एक IoT एप्लिकेशन के अंश](#एक-IoT-एप्लिकेशन-के-अंश)
```
Comment (EliteWarrior315) [inline 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md:25]:
```suggestion
* [एक IoT एप्लिकेशन के अंश](#एक-IoT-एप्लिकेशन-के-अंश)
```

File: 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md [hunk 10]
```diff
@@ -0,0 +1,266 @@
...
+
+किसी भी इंटरनेट कनेक्शन के साथ, सुरक्षा एक महत्वपूर्ण विचार है। एक पुराना मजाक है कि 'आईओटी में एस सुरक्षा के लिए खड़ा है' - आईओटी में कोई 'एस' नहीं है, जिसका अर्थ है कि यह सुरक्षित नहीं है।
+
+IoT डिवाइस क्लाउड सेवा से कनेक्ट होते हैं, और इसलिए केवल उस क्लाउड सेवा की तरह ही सुरक्षित होते हैं - यदि आपकी क्लाउड सेवा किसी डिवाइस को कनेक्ट करने की अनुमति देती है तो दुर्भावनापूर्ण डेटा भेजा जा सकता है, या वायरस के हमले हो सकते हैं। इसके बहुत ही वास्तविक दुनिया के परिणाम हो सकते हैं क्योंकि IoT डिवाइस अन्य उपकरणों को इंटरैक्ट और नियंत्रित करते हैं। उदाहरण के लिए, [स्टक्सनेट वर्म](https://wikipedia.org/wiki/Stuxnet) ने सेंट्रीफ्यूज में वाल्वों को खराब करने के लिए हेरफेर किया। हैकर्स ने [बेबी मॉनिटर तक पहुंचने के लिए खराब सुरक्षा](https://www.npr.org/sections/thetwo-way/2018/06/05/617196788/s-c-mom-says-baby-monitor-was-hacked-experts-say-many-devices-are-vulnerable) का भी फायदा उठाया है और अन्य घरेलू निगरानी उपकरण।
+
+> 💁कभी-कभी IoT डिवाइस और एज डिवाइस डेटा को निजी और सुरक्षित रखने के लिए इंटरनेट से पूरी तरह से अलग नेटवर्क पर चलते हैं। इसे [एयर-गैपिंग](https://wikipedia.org/wiki/Air_gap_(networking)) के नाम से जाना जाता है।
+
+## माइक्रोकंट्रोलर में गहराई से अध्ययन करें
```
Comment (EliteWarrior315) [inline 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md:81]:
```suggestion
## माइक्रोकंट्रोलर में गहराई से अध्ययन
```

File: 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md [hunk 11]
```diff
@@ -0,0 +1,266 @@
...
+[व्याख्यान पूर्व प्रश्नोत्तरी](https://brave-island-0b7c7f50f.azurestaticapps.net/quiz/3)
+
+## परिचय
+
+यह पाठ पिछले पाठ में शामिल कुछ अवधारणाओं में गहराई से अध्ययन करता है।
+
+इस पाठ में हम कवर करेंगे:
+
+* [एक IoT एप्लिकेशन के अंश](#एक-IoT-एप्लिकेशन-के-अंश)
+* [माइक्रोकंट्रोलर्स में गहराई से अध्ययन करें](#माइक्रोकंट्रोलर-में-गहराई-से-अध्ययन-करें)
```
Comment (EliteWarrior315) [inline 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md:26]:
```suggestion
* [माइक्रोकंट्रोलर्स में गहराई से अध्ययन](#माइक्रोकंट्रोलर-में-गहराई- से-अध्ययन)
```

File: 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md [hunk 12]
```diff
@@ -0,0 +1,266 @@
...
+* [सिंगल-बोर्ड कंप्यूटरों में गहराई से अध्ययन करें](#सिंगल-बोर्ड-कंप्यूटर-में-गहराई-से-अध्ययन-करें)
```
Comment (EliteWarrior315) [inline 1-getting-started/lessons/2-deeper-dive/translations/README.hi.md:27]:
```suggestion
* [सिंगल-बोर्ड कंप्यूटर में गहराई से अध्ययन](#सिंगल-बोर्ड-कंप्यूटर-में गहराई-से-अध्ययन)
```

**Review by EliteWarrior315 — APPROVED** [review]

**Review by jimbobbennett — APPROVED** [review]

**Pull request merged**